src/atoms/Table/TableBody.js

```diff
@@ -20,33 +26,78 @@ const theme = createTheme(name, {
   },
 });
 
-const TableBodyTag = createStyledTag(name, (props: *) => ({
-  display: 'grid',
+
+const TableBodyTag = createStyledTag(name, props => ({
+  display: 'flex',
   overflowY: 'auto',
-  position: 'relative',
-  gridAutoRows: 'min-content',
+  flexDirection: 'column',
   borderBottom: `1px solid ${props.theme.COLORS.LIGHT_GRAY1}`,
+  height: '100%',
 }));
 
-function TableBody({
-  children,
-  data,
-  loading,
-  ...rest
-  }: TableBodyProps) {
-  return (
-    <TableBodyTag { ...rest } tagName={ Grid.Layout }>
-      <AsyncContent loading={ loading } stretch>
-        {
-          Array.isArray(data) && typeof children === 'function'
-            ?
-            React.Children.toArray(data.map(children))
-            :
-            children
-        }
-      </AsyncContent>
-    </TableBodyTag>
-  );
+
+class TableBody extends PureComponent<TableBodyProps> {
+
+ hasBodyRows = (children: React$Node) => {
+   let hasBodyRows = false;
+
+   React.Children.forEach(children, child => {
```

Comment:
@ej9x strange check, I think we can delete ability to pass `children` to `TableBoby`, only `data`, after it we can delete this check too

Reply:
why? ability to pass `children` looks useful.